platform/ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -1,33 +1,15 @@
-import React, { useEffect, useRef } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import Typography from '../Typography';
 import { Icons } from '@ohif/ui-next';
 
 const ContextMenu = ({ items, ...props }) => {
-  const contextMenuRef = useRef<HTMLDivElement>(null);
```

Comment:
It was a bit weird that the positioning was being changed here, yet "used" by an ancestor.